extension/jest.config.js

```diff
@@ -3,29 +3,29 @@ module.exports = {
   verbose: true,
   testMatch: [`<rootDir>/test/unit/**/*spec.js`],
   transform: {
-    '^.+\\.(js|jsx)?$': `<rootDir>/node_modules/babel-jest`,
-    '.*\\.vue$': `<rootDir>/node_modules/vue-jest`
+    '.*\\.vue$': `vue-jest`,
+    '^.+\\.(js|jsx)?$': `babel-jest`
```

Comment:
What is the idea here? Are we going then to have a global `node_modules`? Isn't it better to preserve modularity?

Using `yarn workspace` it is so easy. I would go for separate node_modules

Reply:
we are already using yarn workspaces

Reply:
But your idea here is to have these two modules shared by all the subrepos right?

Reply:
I guess though

Reply:
why make an exception here? Better have them always separate except when needed some meta-module, like in [this example](https://github.com/ferreiro/example-monorepo/blob/master/package.json) with concurrently, no?

To keep the same logic

Reply:
is there any problem right now with the logic?

Reply:
well, I would think so by looking at this. But maybe this is the same than with the `rootDir`, just stating `vue-jest`?